services/horizon/internal/integration/protocol14_sponsorship_ops_test.go

```diff
@@ -409,4 +387,200 @@ func TestSponsorData(t *testing.T) {
 		tt.Equal("SponsoredData", dataSponsorshipEffect.DataName)
 	}
 
+	// Test revocation of sponsorship
+	revoke := txnbuild.RevokeSponsorship{
+		SponsorshipType: txnbuild.RevokeSponsorshipTypeData,
+		Data: &txnbuild.DataID{
+			Account:  newAccountPair.Address(),
+			DataName: "SponsoredData",
+		},
+	}
+	txResp, err = itest.SubmitOperations(itest.MasterAccount(), itest.Master(), &revoke)
+	tt.NoError(err)
+	err = xdr.SafeUnmarshalBase64(txResp.ResultXdr, &txResult)
+	tt.NoError(err)
+	tt.Equal(xdr.TransactionResultCodeTxSuccess, txResult.Result.Code)
```

Comment:
I think it can be simplified by using `Must...` method:
```suggestion
	itest.MustSubmitOperations(itest.MasterAccount(), itest.Master(), &revoke)
```